src/constants.js

```diff
@@ -318,6 +318,10 @@ export const SIDEBAR_VIEW_METADATA: 'metadata' = 'metadata';
 export const SIDEBAR_VIEW_ACTIVITY: 'activity' = 'activity';
 export const SIDEBAR_VIEW_VERSIONS: 'versions' = 'versions';
 
+/* ------------------ Content Explorer Grid View ---------------------- */
```

Comment:
We haven't been great about it historically, but we should try to keep constants, types, and utils as close to their components as possible. It doesn't seem like this is meant to be used by any other component/element.